### PR TITLE
5.4.6-rc1 changelog

### PR DESCRIPTION
**Pull request by joshmoore**
Initial changelog that we can rename to 5.4.6 for the final release.

### CODE REVIEW

File: history.txt [hunk 1]
```diff
@@ -5,6 +5,26 @@
...
+----------------------
+
+This is a bug-fix release which also introduces a significant
+new subsystem for read-only operation with which servers can be
+configured to not make changes to the database, the file-system,
+or both. This can be used for scaling up your OMERO installation.
+
+Further improvements include:
+
+- enabled big image support in ImageJ/Fiji
```
Comment (jburel) [inline history.txt]: need extra space after ``-``  for all entries

File: history.txt [hunk 2]
```diff
@@ -5,6 +5,26 @@
...
+- fixed the handling of script params in the web
+- fixed bug in the web History tab
+- fixed CLI login's behavior regarding previous ports
```
Comment (pwalczysko) [inline history.txt]: login's ? Maybe rather use the ...of... construction

File: history.txt [hunk 3]
```diff
@@ -5,6 +5,26 @@
 OMERO version history
 =====================
 
+5.4.6-rc1 (April 2018)
+----------------------
+
+This is a bug-fix release which also introduces a significant
```
Comment (pwalczysko) [inline history.txt]: So, in fact, it is not a bug-fix release in the first place. Start this paragraph with the new feature, then quickly mention ... and some bug-fixes.

File: history.txt [hunk 4]
```diff
@@ -5,6 +5,26 @@
...
+new subsystem for read-only operation with which servers can be
+configured to not make changes to the database, the file-system,
+or both. This can be used for scaling up your OMERO installation.
+
+Further improvements include:
+
+-  enabled big image support in ImageJ/Fiji
+-  fixed the handling of script params in the web
```
Comment (hflynn) [inline history.txt]: 'the web' -> OMERO.web here and below

**Comment by hflynn** [comment]
👍 thanks

**Comment by joshmoore** [comment]
General thumbs-up after the above round of changes.

File: history.txt [hunk 5]
```diff
@@ -5,6 +5,26 @@
 OMERO version history
 =====================
 
+5.4.6-rc1 (April 2018)
+----------------------
+
+This introduces a significant new subsystem for read-only operation
+with which servers can be configured not to make changes to the database,
+the file-system, or both. The goal is to permit the horizontal scaling
```
Comment (mtbc) [inline history.txt]: previously "filesystem" in earlier entries

**Pull request merged**